src/run/formlyNgModelAttrsManipulator.js

```diff
@@ -18,7 +18,7 @@ function addFormlyNgModelAttrsManipulator(formlyConfig, $interpolate) {
       return template;
     }
     el.innerHTML = template;
-    var modelNodes = el.querySelectorAll('[ng-model], [data-ng-model]');
+    var modelNodes = el.querySelectorAll('[ng-model]:not([formly-skip-ng-model-attrs-manipulator]):not(.ignored-thing), [data-ng-model]:not([formly-skip-ng-model-attrs-manipulator]):not(.ignored-thing)');
```

Comment:
So how do I pull your changes? 

git pull https://github.com/formly-js/angular-formly.git master

I tried the above and got conflicts with the test file which I resolved and then it told me to do a rebase --continue which I did but looks like my files keep going back to what they were before.